### PR TITLE
Update reference to `paragonie/random_compat` in composer.json

### PR DESCRIPTION
**Pull request by I-Valchev**

### CODE REVIEW

**Comment by bobdenotter** [comment]
Ah, great.. seems like it's breaking because of this: https://github.com/nelmio/NelmioSecurityBundle/pull/236

(my guess is that the `replace` in a dependency doesn't 'bubble up' for a dependency added in the repo itself..)

File: composer.json [hunk 1]
```diff
@@ -15,9 +15,6 @@
         "nelmio/security-bundle": "^2.10",
         "symfony/flex": "^1.6"
     },
```
Comment (I-Valchev) [inline composer.json:17]:
```suggestion
    },
    "replace": {	
        "paragonie/random_compat": "*"	
    },
```

File: composer.json [hunk 2]
```diff
@@ -15,8 +15,8 @@
...
-    "replace": {
-        "paragonie/random_compat": "9.99.99"
+    "replace": {
```
Comment (I-Valchev) [inline composer.json:18]:
```suggestion
    "replace": {
```

File: composer.json [hunk 3]
```diff
@@ -16,7 +16,7 @@
         "symfony/flex": "^1.6"
     },
     "replace": {
-        "paragonie/random_compat": "9.99.99"
+        "paragonie/random_compat": "*"
```
Comment (I-Valchev) [inline composer.json:19]:
```suggestion
        "paragonie/random_compat": "*"
```

**Comment by bobdenotter** [comment]
After this lands, tests should pass again: https://github.com/bolt/core/pull/2303

**Pull request merged**